src/index.html

```diff
@@ -17,16 +17,34 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="/submit-your-form-handler"
+      method="post"
+      class="search-form"
+    >
+      <div class="search-group">
+        <div
+          class="search"
+          data-qa="big"
+        >
+          <input
+            type="text"
+            class="search__bar search__bar--top"
+            data-qa="keypress"
+            placeholder="Try “Los Angeles“"
+          />
+        </div>
+        <div
+          class="search"
+          data-qa="small"
+        >
+          <input
+            type="text"
+            class="search__bar search__bar--bottom"
+            placeholder="Try “Los Angeles“"
```

Comment:
The attributes in the `<input>` tag should start each on a new line with 2-space indentation related to the tag to keep the attributes correctly formatted. The tag’s closing bracket should be on the same level as the opening one.